[hi] Add content/hi/docs/home/_index.md

verma-kunal opened the pull request:
This PR is a part of ongoing `hi` localization effort

/language hi

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,67 @@
+---
+approvers:
```
```suggestion

```

verma-kunal replied:
oh yes!! I saw this & had left that for the last !!
Thanks for pointing it out 👍

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,67 @@
+---
+approvers:
+- chenopis
```
```suggestion

```

anubha-v-ardhan reviewed (changes requested):
Comparaing with `ko` localization, I found we can remove `approvers:`

verma-kunal commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,67 @@
+cid: docsHome
+layout: docsportal_home
+class: gridPage gridPageHome
+linkTitle: "Home"  # to do
```
@anubha-v-ardhan needed your suggestions here!! what can `Home` be replaced with in Hindi
I was a bit confused in that

anubha-v-ardhan replied:
`होम`, the `ko` localization has used `홈` which is basically the exact sound of `home`, just written in their respective script. We can do the same for these type of problems.

verma-kunal replied:
alright! done that 👍

mittalyashu commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+hide_feedback: true
+menu:
+  main:
+    title: "प्रलेखन "
```
```suggestion
    title: "प्रलेखन"
```

mittalyashu replied:
I am not sure if they changed their docs guidelines recently, but you can also try removing the `"`.

verma-kunal commented:
@mittalyashu , actually i was referring the `Korean` Localization & according to that did the necessary changes

mittalyashu commented:
After taking a took at english doc, I think it's best to remove the quotes, as it will make the content readable more easily from codebase.

kodeninja commented:
IMO, "कुबेरनेट्स" does not match `kubernetes` (in spelling or pronunciation). Perhaps, something like "कूबरनेटिस"?

Also, it would be great if we could at least _attempt_ to provide the pure Hindī terms for things like "container", "pod", "engine", "orchestration" and so on, rather than just writing the english words in dévanāgarī.

Hopefully, someone with enough Sanskrit knowledge can help come up with appropriate terms for, at least, the important concepts 🤷🏼‍♂️.

verma-kunal commented:
> After taking a took at english doc, I think it's best to remove the quotes, as it will make the content readable more easily from codebase.

which quotes are you referring to here? can you pls point that out !!

verma-kunal commented:
> IMO, "कुबेरनेट्स" does not match `kubernetes` (in spelling or pronunciation). Perhaps, something like "कूबरनेटिस"?
> 
> Also, it would be great if we could at least _attempt_ to provide the pure Hindī terms for things like "container", "pod", "engine", "orchestration" and so on, rather than just writing the english words in dévanāgarī.
> 
> Hopefully, someone with enough Sanskrit knowledge can help come up with appropriate terms for, at least, the important concepts 🤷🏼‍♂️.
> 
> https://www.learnsanskrit.cc/ can perhaps be used to suggest some options.

I get your point there! Maybe we can find certain words & add those to the sheet here: https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0
for ease of use!!

kodeninja commented:
@verma-kunal, thanks! I added some comments to that google doc, and also sent a request to become editor, if that's ok.

Also, I added some terms in the main PR under https://github.com/kubernetes/website/issues/29353#issuecomment-898740447. You can add those to the doc as well.

I follow some twitter handles like [@vakibs](https://twitter.com/vakibs) who are polyglots, and very knowledgeable in Indic languages. We can take their help/guidance in coming up the appropriate Hindī terms for the core concepts.

The best part is that the same terms, since they come from Sanskrit, can be used in translations in other Indic languages as well.

anubha-v-ardhan commented:
> IMO, "कुबेरनेट्स" does not match `kubernetes` (in spelling or pronunciation). Perhaps, something like "कूबरनेटिस"?
> 
> Also, it would be great if we could at least _attempt_ to provide the pure Hindī terms for things like "container", "pod", "engine", "orchestration" and so on, rather than just writing the english words in dévanāgarī.
> 
> Hopefully, someone with enough Sanskrit knowledge can help come up with appropriate terms for, at least, the important concepts 🤷🏼‍♂️.
> 
> https://www.learnsanskrit.cc/ can perhaps be used to suggest some options.

@kodeninja We appreciate your suggestions but in my view the Kubernetes objects should be written the same as in English, just in Devnagri script. The goal of localization is to help folks understand the docs in Hindi and not word to word pure translation. Hope you understand.

verma-kunal commented:
> @verma-kunal, thanks! I added some comments to that google doc, and also sent a request to become editor, if that's ok.
> 
> Also, I added some terms in the main PR under [#29353 (comment)](https://github.com/kubernetes/website/issues/29353#issuecomment-898740447). You can add those to the doc as well.
> 
> I follow some twitter handles, like [@vakibs](https://twitter.com/vakibs), who are polyglots, and very knowledgeable in Indic languages. We can take their help/guidance in coming up the appropriate Hindī terms for the core concepts.
> 
> The best part is that the same terms, since they come from Sanskrit, can be used in translations in other Indic languages as well.

As mentioned by @anubha-v-ardhan , the aim of this localization is not to have `pure word to word translation` of all the terms but to have the docs in Hindi for ease of understanding!! 
Just simple translations would do the work as well, Hope you understand!!
Thank you for the suggestions & feedbacks. Would love to see you around contributing to the initiative 🙂

sftim commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  title: प्रलेखन में योगदान करें 
+  description: कोई भी योगदान दे सकता है, चाहे आप प्रोजेक्ट में नए हों या आप लंबे समय से हैं|
+  button: प्रलेखन में योगदान करें 
+  button_path: /hi/docs/contribute
```
```suggestion
  button_path: /docs/contribute
```
unless `/hi/docs/contribute` is already there to link to.

verma-kunal replied:
Oh, actually i was following the `Korean` localization, so according to that i edited the link!
Sure, we can remove that for now 🙂

sftim commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  title: "कुबेरनेट्स का उपयोग करना सीखें"
+  description: "सामान्य कार्यों को देखें और चरणों के एक छोटे क्रम का उपयोग करके उन्हें करें|"
+  button: "कार्य देखें"
+  button_path: "/hi/docs/tasks"
```
To make this work, you'll need to localize `/docs/tasks` (which is OK, but the link target needs to exist).

verma-kunal replied:
we'll be localization that file as well, right, in the near future !! So, for the time-being, should we remove this or keep that as it is (again, taking ref. from `Korean `localization), but i do get your point here 🤔

sftim replied:
```suggestion
  button_path: "/docs/tasks"
```
_for now_

sftim commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+- name: release-notes
+  title: K8s रिलीज नोट्स 
+  description: यदि आप कुबेरनेट्स स्थापित कर रहे हैं या नवीनतम संस्करण में अपग्रेड कर रहे हैं, तो वर्तमान रिलीज़ नोट्स देखें|
+  button: "Download Kubernetes"
```
Perhaps use डाउनˈलअउड्/ instead of download?

verma-kunal replied:
Done! Localized the whole text now

sftim reviewed (commented):
Wow, there's good steps being made here. Here's some feedback on what I see at the moment.

mittalyashu commented:
@verma-kunal Can you remove double quotes from the frontmatter in markdown?

verma-kunal commented:
> @verma-kunal Can you remove double quotes from the frontmatter in markdown?

@mittalyashu can you pls specify like from which parts I have to remove the `double quotes`!! that would be helpful

mittalyashu commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+- name: release-notes
+  title: K8s रिलीज नोट्स 
+  description: यदि आप कुबेरनेट्स स्थापित कर रहे हैं या नवीनतम संस्करण में अपग्रेड कर रहे हैं, तो वर्तमान रिलीज़ नोट्स देखें|
+  button: "डाउनलोड कुबेरनेट्स"
```
```suggestion
  button: डाउनलोड कुबेरनेट्स
```

for example

sftim replied:
Either is fine; YAML doesn't mind.

mittalyashu commented:
/assign verma-kunal

anubha-v-ardhan commented:
/retitile [hi] Add content/hi/docs/home/_index.md

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button_path: "/hi/docs/concepts"
+- name: tutorials
+  title: "कुबेरनेट्स को उपयोग कर ने का प्रयास करें"
+  description: "कुबेरनेट्स में एप्लिकेशन को परिनियोजित करने का तरीका जानने के लिए ट्यूटोरियल का अनुसरण करें|"
```
```suggestion
  description: "कुबेरनेट्स में एप्लिकेशन को डिप्लॉय करने के लिए ट्यूटोरियल का अनुसरण करें|"
```

verma-kunal replied:
i think till then, we had not decided a global word for this!! so, thats why wasn't able to make changes
Thanks

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button: "ट्यूटोरियल देखें"
+  button_path: "/hi/docs/tutorials"
+- name: setup
+  title: "एक कुबेरनेट्स क्लस्टर सेट करें"
```
```suggestion
  title: "एक कुबेरनेट्स क्लस्टर सेटअप करें"
```

anubha-v-ardhan replied:
cc: @verma-kunal
If you agree with the changes, please commit these suggestions :)

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+- name: setup
+  title: "एक कुबेरनेट्स क्लस्टर सेट करें"
+  description: "अपने संसाधनों और जरूरतों के आधार पर कुबेरनेट्स को चालू करें|"
+  button: "कुबेरनेट्स को सेट करें "
```
```suggestion
  button: "कुबेरनेट्स सेटअप करें "
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button_path: "/hi/docs/setup"
+- name: tasks
+  title: "कुबेरनेट्स का उपयोग करना सीखें"
+  description: "सामान्य कार्यों को देखें और चरणों के एक छोटे क्रम का उपयोग करके उन्हें करें|"
```
```suggestion
  description: "चरणों के एक छोटे क्रम का अनुसरण करके सामान्य कार्यों को करें।"
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button_path: "/hi/docs/tasks"
+- name: training
+  title: "प्रशिक्षण"
+  description: "कुबेरनेट्स में प्रमाणित हो जाएं और अपने क्लाउड नेटिव प्रोजेक्ट को सफल बनाएं!"
```
```suggestion
  description: "कुबेरनेट्स में प्रमाणित हो कर अपने क्लाउड नेटिव प्रोजेक्ट सफल बनाएं!"
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button_path: "/training"
+- name: reference
+  title: संदर्भ जानकारी देखें 
+  description: शब्दावली, कमांड लाइन सिंटैक्स, एपीआई संसाधन प्रकार और सेटअप टूल दस्तावेज़ खोजें|
```
```suggestion
  description: शब्दावली, कमांड लाइन सिंटैक्स, API संसाधनो के प्रकार और टूल सेटअप करने के प्रलेखन। 
```

verma-kunal replied:
done manually!!

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  title: संदर्भ जानकारी देखें 
+  description: शब्दावली, कमांड लाइन सिंटैक्स, एपीआई संसाधन प्रकार और सेटअप टूल दस्तावेज़ खोजें|
+  button: संदर्भ देखें 
+  button_path: /hi/docs/reference
```
```suggestion
  button_path: /docs/reference
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button_path: /hi/docs/reference
+- name: contribute
+  title: प्रलेखन में योगदान करें 
+  description: कोई भी योगदान दे सकता है, चाहे आप प्रोजेक्ट में नए हों या आप लंबे समय से हैं|
```
```suggestion
  description: कोई भी योगदान दे सकता है, चाहे आप प्रोजेक्ट में नए हैं या आप लंबे समय से हैं|
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button: प्रलेखन में योगदान करें 
+  button_path: /docs/contribute
+- name: release-notes
+  title: K8s रिलीज नोट्स 
```
```suggestion
  title: कुबेरनेट्स रिलीज नोट्स 
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button_path: /docs/contribute
+- name: release-notes
+  title: K8s रिलीज नोट्स 
+  description: यदि आप कुबेरनेट्स स्थापित कर रहे हैं या नवीनतम संस्करण में अपग्रेड कर रहे हैं, तो वर्तमान रिलीज़ नोट्स देखें|
```
```suggestion
  description: यदि आप कुबेरनेट्स इंस्टॉल कर रहे हैं या नवीनतम संस्करण में अपग्रेड कर रहे हैं, तो वर्तमान रिलीज़ नोट्स देखें|
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+- name: release-notes
+  title: K8s रिलीज नोट्स 
+  description: यदि आप कुबेरनेट्स स्थापित कर रहे हैं या नवीनतम संस्करण में अपग्रेड कर रहे हैं, तो वर्तमान रिलीज़ नोट्स देखें|
+  button: "डाउनलोड कुबेरनेट्स"
```
```suggestion
  button: "कुबेरनेट्स डाउनलोड करें"
```

verma-kunal replied:
done manually !!

anubha-v-ardhan reviewed (changes requested):
Final Review : )
Please point out if I have still missed some inaccuracy.

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button_path: "/docs/setup/release/notes"
+- name: about
+  title: प्रलेखन के बारे में
+  description: इस वेबसाइट में कुबेरनेट्स के वर्तमान और पिछले 4 संस्करणों के लिए दस्तावेज़ शामिल हैं|
```
```suggestion
  description: इस वेबसाइट में कुबेरनेट्स के वर्तमान और पिछले 4 संस्करणों के लिए प्रलेखन हैं|
```

sftim commented:
/label tide/merge-method-squash

Feel free to remove this label (or request removal) if you squash those commits yourself, @verma-kunal

sftim commented:
When that label is set, Prow tells GitHub to squash the PR's commits onto the target branch, rather than doing a merge.

It's still slightly tidier if the author squashes their commits, rather than setting the label and leaving it to GitHub. Nevertheless, using the label-based squashing is fine.

anubha-v-ardhan commented:
Got it, Thanks!

verma-kunal commented:
> When that label is set, Prow tells GitHub to squash the PR's commits onto the target branch, rather than doing a merge.
> 
> It's still slightly tidier if the author squashes their commits, rather than setting the label and leaving it to GitHub. Nevertheless, using the label-based squashing is fine.

Oh! got it!! So, in the future PRs, should we follow the label-squashing approach or user squashing their commits?

verma-kunal commented:
> /label tide/merge-method-squash
> 
> Feel free to remove this label (or request removal) if you squash those commits yourself, @verma-kunal

got it!!

anubha-v-ardhan commented:
@verma-kunal Author squashing commit is tidier, so we will prefer it.

verma-kunal commented:
Squashed the commits!! Please do have a look 🙂

remove /label tide/merge-method-squash

verma-kunal commented:
remove tide/merge-method-squash

verma-kunal commented:
remove-tide/merge-method-squash

anubha-v-ardhan commented:
@verma-kunal Some suggestions were pending to be incorporated. We generally squash commits once the PR is finalised to be merged. 
However, you can implement the suggestions in one more commit, we can merge 2 commits without squash.

verma-kunal commented:
> @verma-kunal Some suggestions were pending to be incorporated. We generally squash commits once the PR is finalised to be merged.
> Can you look at the suggestions and implement the changes.

i actually commited those changes manually!! can you please check?

anubha-v-ardhan commented:
The changes seem to be hidden due to too many conversations. You can go to files changed tab to see them. There are suggestions left

verma-kunal commented:
> The changes seem to be hidden due to too many conversations. You can go to files changed tab to see them. There are suggestions left

sure, would check these!!

verma-kunal commented:
@anubha-v-ardhan , is this good to go? 🙂

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+cards:
+- name: concepts
+  title: "मूल बातें समझें"
+  description: "कुबेरनेट्स और इसकी मूलभूत अवधारणाओं के बारे में जानें|"
```
```suggestion
  description: "कुबेरनेट्स और इसकी मूलभूत अवधारणाओं के बारे में जानें।"
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button_path: "/docs/concepts"
+- name: tutorials
+  title: "कुबेरनेट्स उपयोग करने का प्रयास करें"
+  description: "कुबेरनेट्स में एप्लिकेशन को डिप्लॉय करने के लिए ट्यूटोरियल का अनुसरण करें|"
```
```suggestion
  description: "कुबेरनेट्स में एप्लिकेशन को डिप्लॉय करने के लिए ट्यूटोरियल का अनुसरण करें।"
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button_path: "/hi/docs/tutorials"
+- name: setup
+  title: "एक कुबेरनेट्स क्लस्टर सेटअप करें"
+  description: "अपने संसाधनों और जरूरतों के आधार पर कुबेरनेट्स को चालू करें|"
```
```suggestion
  description: "अपने संसाधनों और जरूरतों के आधार पर कुबेरनेट्स को चालू करें।"
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button_path: /docs/reference
+- name: contribute
+  title: प्रलेखन में योगदान करें 
+  description: कोई भी योगदान दे सकता है, चाहे आप प्रोजेक्ट में नए हैं या आप लंबे समय से हैं|
```
```suggestion
  description: कोई भी योगदान दे सकता है, चाहे आप प्रोजेक्ट में नए हैं या आप लंबे समय से हैं।
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button_path: /docs/contribute
+- name: release-notes
+  title: कुबेरनेट्स रिलीज नोट्स 
+  description: यदि आप कुबेरनेट्स इंस्टॉल कर रहे हैं या नवीनतम संस्करण में अपग्रेड कर रहे हैं, तो वर्तमान रिलीज़ नोट्स देखें|
```
```suggestion
  description: यदि आप कुबेरनेट्स इंस्टॉल कर रहे हैं या नवीनतम संस्करण में अपग्रेड कर रहे हैं, तो वर्तमान रिलीज़ नोट्स देखें।
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+  button_path: "/docs/setup/release/notes"
+- name: about
+  title: प्रलेखन के बारे में
+  description: इस वेबसाइट में कुबेरनेट्स के वर्तमान और पिछले 4 संस्करणों के लिए प्रलेखन हैं|
```
```suggestion
  description: इस वेबसाइट में कुबेरनेट्स के वर्तमान और पिछले 4 संस्करणों के लिए प्रलेखन हैं।
```

anubha-v-ardhan reviewed (commented):
Please note that `।` and `|` are different characters and we should prefer `।`

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,65 @@
+description: >
+  कुबेरनेट्स कंटेनरीकृत एप्लीकेशन के परिनियोजन, स्केलिंग और प्रबंधन को स्वचालित करने के लिए एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेशन इंजन है| यह ओपन सोर्स प्रोजेक्ट Cloud Native Computing Foundation द्वारा होस्ट किया गया है।
+overview: >
+  कुबेरनेट्स कंटेनरीकृत एप्लीकेशन के परिनियोजन, स्केलिंग और प्रबंधन को स्वचालित करने के लिए एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेशन इंजन है। यह ओपन सोर्स प्रोजेक्ट Cloud Native Computing Foundation द्वारा होस्ट किया गया है|(<a href="https://www.cncf.io/about">CNCF</a>).
```
```suggestion
  कुबेरनेट्स कंटेनरीकृत एप्लीकेशन के डिप्लॉयमेंट, स्केलिंग और प्रबंधन को स्वचालित करने के लिए एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेशन इंजन है। यह ओपन सोर्स प्रोजेक्ट Cloud Native Computing Foundation द्वारा होस्ट किया गया है|(<a href="https://www.cncf.io/about">CNCF</a>).
```

anubha-v-ardhan reviewed (commented):
Can you squash the commits please so we can merge this.

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,69 @@
+  button_path: "/hi/docs/tutorials"
+- name: setup
+  title: "एक कुबेरनेट्स क्लस्टर सेटअप करें"
+<<<<<<< HEAD
```
```suggestion
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,69 @@
+  title: "एक कुबेरनेट्स क्लस्टर सेटअप करें"
+<<<<<<< HEAD
+  description: "अपने संसाधनों और जरूरतों के आधार पर कुबेरनेट्स को चालू करें|"
+=======
```
```suggestion
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,69 @@
+  description: "अपने संसाधनों और जरूरतों के आधार पर कुबेरनेट्स को चालू करें|"
+=======
+  description: "अपने संसाधनों और जरूरतों के आधार पर कुबेरनेट्स को चालू करें।"
+>>>>>>> 997deca04a445610cc54d20e55ac8ab5f87003d0
```
```suggestion
```

anubha-v-ardhan commented on content/hi/docs/home/_index.md:
```diff
@@ -0,0 +1,69 @@
+  description: "कुबेरनेट्स में एप्लिकेशन को डिप्लॉय करने के लिए ट्यूटोरियल का अनुसरण करें।"
+  button: "ट्यूटोरियल देखें"
+  button_path: "/hi/docs/tutorials"
+- name: setup
+  title: "एक कुबेरनेट्स क्लस्टर सेटअप करें"
+<<<<<<< HEAD
+  description: "अपने संसाधनों और जरूरतों के आधार पर कुबेरनेट्स को चालू करें|"
+=======
+  description: "अपने संसाधनों और जरूरतों के आधार पर कुबेरनेट्स को चालू करें।"
+>>>>>>> 997deca04a445610cc54d20e55ac8ab5f87003d0
+  button: "कुबेरनेट्स सेटअप करें "
+  button_path: "/hi/docs/setup"
```
```suggestion
- name: setup
  title: "एक कुबेरनेट्स क्लस्टर सेटअप करें"
  description: "अपने संसाधनों और जरूरतों के आधार पर कुबेरनेट्स को चालू करें।"
  button: "कुबेरनेट्स सेटअप करें "
  button_path: "/hi/docs/setup"
```

verma-kunal replied:
i wonder where that all came from? 🤔

anubha-v-ardhan reviewed (commented):
LGTM
You can squash the commits now!

verma-kunal commented:
> LGTM
> You can squash the commits now!

there are only 2 commits after the previous squash!! do we need still need to do a squash here?

anubha-v-ardhan commented:
Sorry, but seems like we have 16 commits here. Maybe the previous squash failed or for some reason.

anubha-v-ardhan commented:
We can merge using `tide/merge-method-squash` if there's an issue.

verma-kunal commented:
> We can merge using `tide/merge-method-squash` if there's an issue.

no worries at all !! i can do that again 🙂

verma-kunal commented:
Done!! is this good to go?

anubha-v-ardhan commented:
/remove-tide/merge-method-squash

anubha-v-ardhan commented:
/remove-label tide/merge-method-squash

anubha-v-ardhan commented:
/lgtm
/approve

Thank you @verma-kunal
